### PR TITLE
Fixed run mojo to use services param in standard env app check #312

### PR DESCRIPTION
**Pull request by matt-blanchette**

### CODE REVIEW

File: src/test/java/com/google/cloud/tools/maven/RunMojoTest.java [hunk 1]
```diff
@@ -50,6 +51,11 @@
 @RunWith(JUnitParamsRunner.class)
 public class RunMojoTest extends AbstractDevServerTest {
 
+  private static final String STANDARD_PROJECT_WEBAPP =
```
Comment (loosebazooka) [inline src/test/java/com/google/cloud/tools/maven/RunMojoTest.java]: just need to add a negative test here, where it fails (one standard app, one non-standard app)

Comment (matt-blanchette) [reply src/test/java/com/google/cloud/tools/maven/RunMojoTest.java]: 👍

**Comment by matt-blanchette** [comment]
Any way to re-trigger these checks?

**Comment by loosebazooka** [comment]
looks good, thanks for your contributions!

**Pull request merged**